### ISSUE TITLE
Crashes after updating to 83.0.4103.116

### ISSUE THREAD

**Issue by ACheshirov** [Eloston/ungoogled-chromium#1089]
**Describe the bug**
After updating (from PPA) to 83.0.4103.116 version it just start randomly crashed. There is no error or something like that, the browser just stops.

**To Reproduce**
I can't reproduce it because it crashes in random intervals.

**Environment (please complete the following information):**
 - OS: Ubuntu 18.04.4 LTS x86_64
 - Kernel: 5.3.0-61-generic
 - DE: KDE
 - WM: KWin
 - Theme: Breeze Dark [KDE], Breeze [GTK2/3]
 - CPU: AMD Ryzen 7 3700U with Radeon Vega Mobile Gfx (8) @ 2.300GHz
 - GPU: AMD Picasso
 - ungoogled-chromium version: 83.0.4103.116

**Additional context**
Actually this happen to my both devices with different hardware - a laptop and PC.

I have a console error (I have started the browser with the "--enable-logging --v=1" command:

> [12004:12027:0625/160726.374271:FATAL:ref_counted.cc(66)] Check failed: --ref_count_ != std::numeric_limits<decltype(ref_count_)>::max(). 
#0 0x5608b869b1a9 (/usr/lib/chromium/chromium+0x440a1a8)

Received signal 6
#0 0x5608b869b1a9 (/usr/lib/chromium/chromium+0x440a1a8)
  r8: 0000000000000000  r9: 00007fe3b48fe070 r10: 0000000000000008 r11: 0000000000000246
 r12: 000000000000008a r13: 00007fe3b48ff340 r14: 00002ac9909fe008 r15: 00007fe3b48ff338
  di: 0000000000000002  si: 00007fe3b48fe070  bp: 00007fe3b48fe2c0  bx: 00007fe3b48fe324
  dx: 0000000000000000  ax: 0000000000000000  cx: 00007fe3ca3e3e97  sp: 00007fe3b48fe070
  ip: 00007fe3ca3e3e97 efl: 0000000000000246 cgf: 002b000000000033 erf: 0000000000000000
 trp: 0000000000000000 msk: 0000000000000000 cr2: 0000000000000000
[end of stack trace]
Calling _exit(1). Core file will not be generated.

**Comment by jstkdng** [comment]
I think the ppa has a package containing the debugging symbols, install that, replicate the issue and submit the logs, they should have more information than just numbers.

**Comment by twielfaert** [comment]
Same here with the latest Ubuntu 20.04 package

Debugging symbols doesn't seem to be included in the OBS builds?

**Comment by ACheshirov** [comment]
I don't know how to get that "debugging symbols". I have installed these additional dbgsym ddeb packages (if this is it) and now what?
My browser is still randomly crashing (and I can't find any crash logs or something) and it's not usable, so I had to go back to chrome temporarily.

**Comment by braewoods** [comment]
Given what limited information that exists, it appears the crash is due to a failed assertion. The code is checking the result of a decrement of a reference counter that happens to cause underflow back to the unsigned maximum value. This means there's either an infinite cycle causing it to decrement the reference counter too much or just a plain old double free bug of sorts.

Either way it's not clear what is triggering this or whether it's even related to our patches.

**Comment by ACheshirov** [comment]
I'm a developer (web) myself and I know what is it when a someone is telling you that something is broken but without any specific information. I'm sorry but I don't know how to get enough data about this bug.
I can tell for sure that I'm using now Google Chrome, same version (83.0.4103.116) and there is no any crashes.

Crashes appears when I'm typing something in the address bar, sometimes when I'm watching a video clip in YT. It's look like they are on random events so it's hard to catch them up.

**Comment by braewoods** [comment]
@twielfaert You're having a similar crash to @ACheshirov? Could you put up more system information like they have provided? I'm wondering if this could be a hardware related issue. Chromium has been known to have issues on some hardware platforms but not others. Though I can't see why that would be the case here due to the nature of the error message.

**Comment by ACheshirov** [comment]
My PC specs:
OS: Ubuntu 20.04 LTS x86_64
Host: MS-7A34 2.0
Kernel: 5.4.0-39-generic
DE: Plasma
WM: KWin
Theme: Breeze Dark [Plasma], Breeze [GTK2/3]
CPU: AMD Ryzen 5 1600X (12) @ 3.600GHz
GPU: NVIDIA GeForce GTX 1060 3GB
Memory: 5461MiB / 16027MiB

**Comment by twielfaert** [comment]
@braewoods Yes indeed.

> OS: Ubuntu 20.04 LTS x86_64
> Kernel: 5.4.0-39-generic
> Host: Dell XPS 13 9370/0F6P3V, BIOS 1.12.1 12/11/2019
> DE: Gnome 3.36.2
> WM: Gnome Shell
> Theme: Yaru
> CPU: Intel Core i7-8550U CPU @ 1.80GHz
> GPU: Mesa Intel UHD Graphics 620 (KBL GT2)
> Memory: 15.3 GiB

**Comment by braewoods** [comment]
Before we go any further I would ask both of you to try using it with a fresh profile and see if the problem persists. Our chromium has been known to have reliability issues when used with another chromium's profile.

**Comment by cypherpunk1984** [comment]
Same thing happening to me. Linux Mint 19.3 Cinnamon 64 bits. Random crashes after update to the mentioned version, as OP said there is no way to reproduce it because it is so random, even sometimes when I'm not using the pc, chromium just shuts down and have to open it again and restore the tabs if there were any opened. At least 10 times in the last 24 hs, super annoying.

**Comment by jstkdng** [comment]
@cypherpunk1984 what are the specifications or your pc?

@ACheshirov 
now that you've installed the symbols, run chromium with those flags: `--enable-logging --v=1` and if a crash appears again comment the output, that will really help to figure out what the problem is.

**Comment by cypherpunk1984** [comment]
> @cypherpunk1984 what are the specifications or your pc?
> 
> @ACheshirov
> now that you've installed the symbols, run chromium with those flags: `--enable-logging --v=1` and if a crash appears again comment the output, that will really help to figure out what the problem is.

It's a Lenovo Yoga 900 13ISK2
Intel Core i7 6560U 2.20Ghz
16gb RAM
Video Intel Iris 540

Previous versions were running perfectly.

**Comment by wchen342** [comment]
I had a quick look at the function mentioned. The line assertion failed is in `base/memory/ref_counted.cc` which is a safety module for memory options. The comment for the function reads:
```
void RefCountedBase::ReleaseImpl() const {
  // Make sure the subtraction didn't wrap back around from 0 to the max value.
  // That could cause memory leaks, and may induce application-semantic
  // correctness or safety bugs. (E.g. what if we really needed that object to
  // be destroyed at the right time?)
  //
  // Note that unlike with overflow, underflow could also happen on 32-bit
  // architectures. Arguably, we should do this check on32-bit machines too.
  CHECK(--ref_count_ != std::numeric_limits<decltype(ref_count_)>::max());
}
```

A quick `grep` shows this function is only used in `blink`, all under `third_party/blink/renderer`. The affected functions are mainly related to some text processing, including WTF and layouts. The most likely suspect is something called `ParkableString`.

Hopefully this can help you guys find the problem.

**Comment by jstkdng** [comment]
Really weird for this to only affect ubuntu and possibly debian but arch is running smoothly.
I checked the debian/ubuntu build flags and they aren't using `is_official_build=true`, that could be the issue as CHECKS shouldn't be run on builds for end users IIRC.

**Comment by wchen342** [comment]
> they aren't using is_official_build=true, that could be the issue as CHECKS shouldn't be run on builds for end users

I think what you mentioned is `DCHECK()` which should only exist in debug builds? But the `is_official_build` flag may have other unexpected effects since it enables extra optimization flags in `gn`.

**Comment by Eloston** [comment]
> I think what you mentioned is DCHECK() which should only exist in debug builds?

I think @jstkdng is referring to the `CHECK()` macro that's used in your comment above: https://github.com/Eloston/ungoogled-chromium/issues/1089#issuecomment-650470828

> I checked the debian/ubuntu build flags and they aren't using is_official_build=true, that could be the issue as CHECKS shouldn't be run on builds for end users IIRC.

This doesn't make sense to me. The point of the `CHECK`s are to ensure that the program execution satisfies the invariants defined by the programmers. They're removed in official builds purely as a performance boost; just because the program no longer crashes does not mean the program is running correctly; a whole host of awful things can happen. You don't have to take my word for it, even the comment of `RefCountedBase::ReleaseImpl()` warns as much: `That could cause memory leaks, and may induce application-semantic correctness or safety bugs.`. This isn't just bad for your own browsing; it hurts everyone else and damages the trust we have with users when something bad does happen.

Anyway, this rant was a bit uncalled for so let's just keep it at that. Hopefully my message makes sense.

**Comment by wchen342** [comment]
> The point of the CHECKs are to ensure that the program execution satisfies the invariants defined by the programmers.

Yes I was trying to say the same thing by saying that only `DCHECK` is supposed to be debug-build-only, `CHECK` can occur on release builds.

**Comment by jstkdng** [comment]
yeah my bad, DCHECK isnt supposed to run on release builds. Then something else in the debian patchset should be the problem.

**Comment by twielfaert** [comment]
> Before we go any further I would ask both of you to try using it with a fresh profile and see if the problem persists. Our chromium has been known to have reliability issues when used with another chromium's profile.

Tried with a new profile, still random crashes.

**Comment by ignmc** [comment]
Hey guys, just wanted to post the message I got in the terminal when running the browser with the `--enable-logging --v=1` flags, in case it can help you:

```
../../third_party/tcmalloc/chromium/src/free_list.h:72] Circular loop in list detected:  0x35825b0d05c0 
Received signal 11 SEGV_MAPERR 000000000039
#0 0x5648652d31a9 (/usr/lib/chromium/chromium+0x440a1a8)
  r8: 0000000000000000  r9: 00005648621beb00 r10: cccccccccccccccd r11: 0000000000000000
 r12: 00007fede2259268 r13: 00007fede2259390 r14: 0000000000000001 r15: 0000000000000001
  di: 0000000000000000  si: 00007fede2259268  bp: 00007fede2259230  bx: 0000000000000069
  dx: 0000000000000069  ax: 0000000000000001  cx: 00007fee02e082b7  sp: 00007fede2259230
  ip: 0000564862893ac4 efl: 0000000000010202 cgf: 002b000000000033 erf: 0000000000000006
 trp: 000000000000000e msk: 0000000000000000 cr2: 0000000000000039
[end of stack trace]
Calling _exit(1). Core file will not be generated.
```
 This is my environment:

```
OS: elementary OS 5.1.5 Hera
Kernel: Linux 4.15.0-108-generic
CPU: Dual-Core Intel® Core™ i7-5500U CPU @ 2.40GHz
GPU: Intel Corporation HD Graphics 5500 (rev 09)
Chromium 83.0.4103.116 built on Ubuntu bionic/18.04, running on elementary hera/5.1.5
```

Hope it helps!

**Comment by numin0us** [comment]
I started getting crashes after updating to the most recent version of ungoogled-chromium as well. Oddly the crash seems to occur at different locations each time, but has been happening fairly regularly. I'll post stack traces as they crop up, I'd be happy to provide further information as well if you tell me what you need to fix the problem. 

```
[2179087:743:0628/160644.198490:VERBOSE1:thread_state.cc(1392)] [state:0x28b8c6b4c840] CollectGarbage: time: 0.76ms stack: NoHeapPointersOnStack marking: AtomicMarking sweeping: EagerSweeping reason: ThreadTerminationGC
Received signal 11 <unknown> 000000000000
#0 0x5611d6fe4919 (/usr/lib/chromium/chromium+0x42c3918)
  r8: 0000000000000000  r9: 0000000000000000 r10: 0000000000004000 r11: 0000000000000000
 r12: 0000000000000000 r13: 000007b646333a70 r14: 000007b6472b6260 r15: 000007b646bfce48
  di: 00005611d6f3ba00  si: 0000000000000000  bp: 00007fd2425331c0  bx: 000007b646bfce20
  dx: 000007b646bfce00  ax: 950ff685e5894855  cx: fffffffd4f715cd3  sp: 00007fd2425331a0
  ip: 00005611d4dc8bc0 efl: 0000000000010206 cgf: 002b000000000033 erf: 0000000000000000
 trp: 000000000000000d msk: 0000000000000000 cr2: 0000000000000000
[end of stack trace]
Calling _exit(1). Core file will not be generated.

$ uname -a
Linux pop-os 5.4.0-7634-generic #38~1591219791~20.04~6b1c5de-Ubuntu SMP Thu Jun 4 02:56:10 UTC 2 x86_64 x86_64 x86_64 GNU/Linux
$ chromium --version
Chromium 83.0.4103.116 built on Ubuntu focal/20.04, running on Pop focal/20.04
$ cat /proc/cpuinfo | grep "model name" | uniq
model name	: Intel(R) Core(TM) i7-9750H CPU @ 2.60GHz
#I'm currently using the NVIDIA card
$ lspci -v  | grep -i 'VGA'
00:02.0 VGA compatible controller: Intel Corporation UHD Graphics 630 (Mobile) (prog-if 00 [VGA controller])
01:00.0 VGA compatible controller: NVIDIA Corporation TU106M [GeForce RTX 2060 Mobile] (rev a1) (prog-if 00 [VGA controller])

WM/DE is i3wm. 
```

**Comment by ObeseChicken** [comment]
I'm also crashing since I updated to this version. Chrome just closes several times a day at random. Ubuntu 18.0.4, kernel 5.3, intel graphics. Definitely real issue, I was expecting to be the only one.

**Comment by lfom** [comment]
More crash logs (Pop!_OS 20.04, kernel 5.4.48-xanmod1/5.4.0-7634-generic, using Gnome on Wayland):

```
00:01:36 chromium: Calling _exit(1). Core file will not be generated.
00:01:36 chromium: Calling _exit(1). Core file will not be generated.
00:01:36 chromium: [end of stack trace]
00:01:36 chromium:  trp: 000000000000000e msk: 0000000000000000 cr2: 0000000000000039
00:01:36 chromium:   ip: 000056428b6913c4 efl: 0000000000010202 cgf: 002b000000000033 erf: 0000000000000006
00:01:36 chromium:   dx: 0000000000000069  ax: 0000000000000001  cx: 00007f23d3cc32cf  sp: 00007f23c48a7f00
00:01:36 chromium:   di: 0000000000000000  si: 00007f23c48a7f38  bp: 00007f23c48a7f00  bx: 0000000000000069
00:01:36 chromium:  r12: 00007f23c48a7f38 r13: 00007f23c48a8060 r14: 0000000000000001 r15: 0000000000000001
00:01:36 chromium:   r8: 0000000000000000  r9: 000056428b025770 r10: cccccccccccccccd r11: 0000000000000000
00:01:36 chromium: #0 0x56428e024919 (/usr/lib/chromium/chromium+0x42c3918)
00:01:36 chromium: Received signal 11 SEGV_MAPERR 000000000039
00:01:36 chromium: ../../third_party/tcmalloc/chromium/src/free_list.h:72] Circular loop in list detected:  0x13feeeafa300
jun 28 22:50:23 chromium: libpng warning: iCCP: profile 'BenQ GW2765': 'RGB ': RGB color space not permitted on grayscale PNG
```

```
01:30:33 chromium: Calling _exit(1). Core file will not be generated.
01:30:33 chromium: Calling _exit(1). Core file will not be generated.
01:30:33 chromium: [end of stack trace]
01:30:33 chromium:  trp: 000000000000000e msk: 0000000000000000 cr2: 0000000300000002
01:30:33 chromium:   ip: 0000000300000002 efl: 0000000000010206 cgf: 002b000000000033 erf: 0000000000000014
01:30:33 chromium:   dx: 0000009719bd5300  ax: 00000097188b1e00  cx: fffffffd4f1ff0bd  sp: 00007f4ed1e6f198
01:30:33 chromium:   di: 000000971846e160  si: 0000000000000000  bp: 00007f4ed1e6f1c0  bx: 0000009719bd5320
01:30:33 chromium:  r12: 0000000000000000 r13: 0000009717b42670 r14: 000000971a1c77e0 r15: 0000009719bd5348
01:30:33 chromium:   r8: 0000000000000000  r9: 0000000000000000 r10: 0000000000004000 r11: 0000000000000000
01:30:33 chromium: #0 0x561c04811919 (/usr/lib/chromium/chromium+0x42c3918)
01:30:33 chromium: Received signal 11 SEGV_MAPERR 000300000002
00:15:01 chromium: Fontconfig error: Cannot load default config file
00:09:48 chromium: [6311:6341:0629/000948.027641:ERROR:trk_protocol_handler.cc(17)] Blocked URL in TrkProtocolHandler: trk:http://update.9oo91eapis.qjz9zk/service/update2/json?cup2key=10:2725199131&cup2hreq=7af25556cbf4b7509a2878c6c1dd963429776d2c15332e38bd0793fd28196036
00:09:48 chromium: [6311:6341:0629/000948.027488:ERROR:url_request.cc(586)] Block URL in URLRequest: http://update.9oo91eapis.qjz9zk/service/update2/json?cup2key=10:2725199131&cup2hreq=7af25556cbf4b7509a2878c6c1dd963429776d2c15332e38bd0793fd28196036
00:09:48 chromium: [6311:6341:0629/000948.023564:ERROR:trk_protocol_handler.cc(17)] Blocked URL in TrkProtocolHandler: trk:https://update.9oo91eapis.qjz9zk/service/update2/json?cup2key=10:2382483762&cup2hreq=7af25556cbf4b7509a2878c6c1dd963429776d2c15332e38bd0793fd28196036
00:09:48 chromium: [6311:6341:0629/000948.023398:ERROR:url_request.cc(586)] Block URL in URLRequest: https://update.9oo91eapis.qjz9zk/service/update2/json?cup2key=10:2382483762&cup2hreq=7af25556cbf4b7509a2878c6c1dd963429776d2c15332e38bd0793fd28196036
```

**Comment by braewoods** [comment]
I've found similar crash reports for the official chromium package from Debian. See here:
https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=963548

Given that their package is experiencing similar issues, I presume the issue lies in something we have in common with their packaging. @Eloston Thoughts?

**Comment by wchen342** [comment]
From the report it looks like an general memory problem. I suspect it has something to do with the allocator. Maybe compile a version with `use_allocator="none"` to test? I don't have a physical Debian machine so I cannot test myself.

**Comment by braewoods** [comment]
@wchen342 what does the Arch packaging use?

**Comment by wchen342** [comment]
The Arch package doesn't experience anything unusual (at least on 106, 116 is just ready today) so it uses the default flags.
I however experienced a strange linking issue when cross compiling which is related to allocator which hint me maybe it can be a problem. My problem was solved by downgrading `lld` though, not sure whether that applies to Debian since Arch has llvm 10.

**Comment by ACheshirov** [comment]
Ok, I found a web page that crashes my browser. I made a video, you can watch it on https://youtu.be/V1N-urqx3tQ
The webpage is: https://dariknews.bg/regioni/Pazardzhik/12-zarazeni-rabotnici-ot-vagonoremontniia-zavod-v-grad-septemvri-2231325

**Comment by PF4Public** [comment]
I use `tcmalloc` for Gentoo too by default. No problems so far.

**Comment by Eloston** [comment]
I think @braewoods is right in https://github.com/Eloston/ungoogled-chromium/issues/1089#issuecomment-650911559, especially considering two non-Debian platforms have no issues.

Closing as invalid because it's an upstream issue. Also pinning for increased visibility since it's a popular-enough issue.

@braewoods if you're able to fix the problem before Debian, feel free to tag a new version.

**Comment by ObeseChicken** [comment]
Heh, well saying "not our bug" is cool and all but due to the new way binaries are released I can't downgrade and I'm stuck with the crashing version.

**Comment by wchen342** [comment]
Isn't it possible to install a specific version using `apt`? Or did OBS remove old builds?

**Comment by ObeseChicken** [comment]
nothing left under force version but the one

**Comment by jstkdng** [comment]
AFAIK, all package managers keep a cache of old binaries unless you remove them manually.

**Comment by ObeseChicken** [comment]
Found other packages but chrome is gone. Used to manually download and keep versions, I have a v79 at latest. This is the shortcomming of the new repo holding only one :( At least it doesn't crash ALL the time, just at random.

**Comment by braewoods** [comment]
New builds were pushed out with the first set of upstream fixes. No guarantees about how it will help with crashing but it may help some as some crashes were likely related to ffmpeg.

**Comment by bcHelix** [comment]
Still getting crashes here after updating unfortunatly.

**Comment by RanTrycl** [comment]
We also have frequent random crashing in 83.0 on two Toshiba Satellites L755 running different versions of Mint (bionic and focal). I'm not good at this but can try to get some crash log info if that would help.

**Comment by lfom** [comment]
Latest version (updated today), it does look like the crash is related to **tcmalloc**:

```
21:48:15 chromium: ../../third_party/tcmalloc/chromium/src/free_list.h:72] Circular loop in list detected:  0x38e2b63bc140
21:48:15 chromium: Received signal 11 SEGV_MAPERR 000000000039
21:48:15 chromium: #0 0x5587ea8fd919 (/usr/lib/chromium/chromium+0x42c3918)
21:48:15 chromium:   r8: 0000000000000000  r9: 00005587e78fe770 r10: cccccccccccccccd r11: 0000000000000000
21:48:15 chromium:  r12: 00007f17412c7f38 r13: 00007f17412c8060 r14: 0000000000000001 r15: 0000000000000001
21:48:15 chromium:   di: 0000000000000000  si: 00007f17412c7f38  bp: 00007f17412c7f00  bx: 0000000000000069
21:48:15 chromium:   dx: 0000000000000069  ax: 0000000000000001  cx: 00007f17506e72cf  sp: 00007f17412c7f00
21:48:15 chromium:   ip: 00005587e7f6a3c4 efl: 0000000000010202 cgf: 002b000000000033 erf: 0000000000000006
21:48:15 chromium:  trp: 000000000000000e msk: 0000000000000000 cr2: 0000000000000039
21:48:15 chromium: [end of stack trace]
21:48:15 chromium: Calling _exit(1). Core file will not be generated.
```
As a side note, it seems that `--disable-dev-shm-usage` makes some difference (less prone to crash).

**Comment by awsms** [comment]
> As a side note, it seems that `--disable-dev-shm-usage` makes some difference (less prone to crash).

That made 0 difference for me unfortunately.

**Comment by andreasmjg** [comment]
I also have this bug - keeps crashing at random intervals. Ubuntu 18.04 LTS

**Comment by noahbliss** [comment]
Dropping on with a +1. Debian system had these crashes with the official debian repo version of chromium as well as this browser.

**Comment by david-lindner** [comment]
I have the same issue with the version installed from PPA on Ubuntu 20.04. But I found as a workaround I don't have any issues when using [the portable linux binary](https://ungoogled-software.github.io/ungoogled-chromium-binaries/releases/linux_portable/64bit/83.0.4103.97-1.1). Did not have any crashes for the past 4 hours, while before I had them every ~10-20 minutes.

**Comment by Fallast** [comment]
it's working for me now, i dont know if the update fixed it for me or if it was this command i tried:
sudo apt purge xserver-xorg-video-intel

**Comment by Fallast** [comment]
> it's working for me now, i dont know if the update fixed it for me or if it was this command i tried:
> sudo apt purge xserver-xorg-video-intel

nevermind, just crashed on me, getting the same error

**Comment by xsmile** [comment]
https://salsa.debian.org/chromium-team/chromium/-/commit/b904fa41d40b967dcc8f6984db52f7a2f6a2c83d might be the solution to this issue. I didn't test it.

**Comment by awsms** [comment]
Looks like the latest update fixed it. I can open https://dariknews.bg/regioni/Pazardzhik/12-zarazeni-rabotnici-ot-vagonoremontniia-zavod-v-grad-septemvri-2231325 without the browser crashing on me.

**Comment by braewoods** [comment]
Latest patches from upstream should now be out in the build. This is probably going to make a real difference for most. If there's no more reports of issues it may be time to unpin this @Eloston.

**Comment by eater** [comment]
Is 83.0.4103.116-1.bionic3 the patched one? That's what I just apt-installed and it crashes.

**Comment by lfom** [comment]
> Is 83.0.4103.116-1.bionic3 the patched one? That's what I just apt-installed and it crashes.

```
chrome://version/
Chromium | 83.0.4103.116 (Developer Build) built on Ubuntu focal/20.04, running on Pop focal/20.04 (64-bit)
Revision | 8f0c18b4dca9b6699eb629be0f51810c24fb6428-refs/branch-heads/4103@{#716}
```
No crashes so far.

**Comment by RanTrycl** [comment]
Everything working smashingly again here on bionic and focal, no crashes now.

**Comment by eater** [comment]
```
# dpkg -i ungoogled-chromium_83.0.4103.116-1.bionic3_amd64.deb
...
$ chromium
[16134:16134:0713/154442.670991:ERROR:sandbox_linux.cc(374)] InitializeSandbox() called with multiple threads in process gpu-process.
[16138:16193:0713/154443.081496:ERROR:url_request.cc(586)] Block URL in URLRequest: https://update.9oo91eapis.qjz9zk/service/update2/json
[16138:16193:0713/154443.081710:ERROR:trk_protocol_handler.cc(17)] Blocked URL in TrkProtocolHandler: trk:https://update.9oo91eapis.qjz9zk/service/update2/json
Received signal 11 SEGV_MAPERR 000000000000
#0 0x557d155a2269 (/usr/lib/chromium/chromium+0x440a268)
  r8: 00000c514ead9170  r9: 0000000000000000 r10: 0000000000000001 r11: 0000000000000000
 r12: 00007f7594deb430 r13: 00000c514ead9170 r14: 00007f7594deb4a8 r15: 00000c514ececc00
  di: 0000000000000000  si: 4f285dd87b802459  bp: 00007f7594deb3a0  bx: 0000000000000000
  dx: 3fa41dc043beccd7  ax: 002f08e989532358  cx: 00007f7594deb4a8  sp: 00007f7594deb360
  ip: 0000557d177b692d efl: 0000000000010206 cgf: 002b000000000033 erf: 0000000000000004
 trp: 000000000000000e msk: 0000000000000000 cr2: 0000000000000000
[end of stack trace]
Calling _exit(1). Core file will not be generated.
```

**Comment by lfom** [comment]
@eater Have you tried to purge the current installation and then add the one from OpensSUSE OBS, that will also offer automatic updates as well?

**Comment by eater** [comment]
@lfom Yes, I did that too but still the same immediate crash.

**Comment by gr01d** [comment]
Can anyone confirm that this is fixed? I plan to update my ungoogled-chromium to the latest version.

**Comment by ignmc** [comment]
@gr01d i've had zero crashes after updating

**Comment by eater** [comment]
Still crashes for me.

**Comment by ACheshirov** [comment]
It doesn't crash on me anymore.

**Comment by gr01d** [comment]
Confirmed Version 83.0.4103.116 (Developer Build)  does not crash on me too.

**Comment by braewoods** [comment]
I'm working on chromium 84. To anyone still experiencing issues, please retest on that once it lands and open a new issue if it persists. This one has dragged on long enough.
